internal/cli/login.go

```diff
@@ -249,12 +249,11 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
 	}
 
 	if resp.StatusCode() == http.StatusTeapot {
-		fmt.Println("Auth is disabled")
 		err = config.Persist(o.ConfigFilePath)
 		if err != nil {
-			return fmt.Errorf("persisting client config: %w", err)
+			return fmt.Errorf("auth is disabled, persisting client config: %w", err)
 		}
-		return nil
+		return fmt.Errorf("Auth is disabled")
```

Comment:
Auth is disabled is not a fatal error and should not return as such. It's just a warning to users (i.e. this happens in a development environment)

If we want to return as error we should tolerate that on the other tests (smoke and e2e/cli)